bin/apisix

```diff
@@ -77,7 +77,12 @@ events {
     accept_mutex off;
     worker_connections {* event.worker_connections *};
     {% if event.use_epoll then %}
+        {% if os_name == "Linux" then %}
     use epoll;
+        {% end %}
+        {% if os_name == "Darwin" then %}
+    use kqueue;
+        {% end %}
```

Comment:
we nend to remove this commit

Reply:
then  we should not provide the configuration for the item "use epoll",  remove it together。